README.md and bootstrap.sh changes

jtktam opened the pull request:
I am using a 5 node coreos cluster and ran into some issues, proposed changes:

1. the ansible_python_interpreter variable setting in the README.md didn't work for me, I had to change it (tested working)
2. I use an offline mirror of the pypy tar file, I made the download URL a variable to easily repoint the location
3. Added some comments to bootstrap.sh

defunctzombie commented:
the only change I like from this is the PYPY_URL variable. The rest isn't needed I think.

jtktam commented:
ansible_python_interpreter="PATH=/home/core/bin:$PATH python"

I am not sure if this was intentional to have a PATH= line inside the quotes, but the sample didn't work for me til i changed this to 

ansible_python_interpreter="/home/core/bin/python"

i agree that the PYPY_URL change and added comments is not needed, but for me it made it easier to understand
